app/models/partners/user.rb

```diff
@@ -32,12 +32,20 @@ class User < Base
 
     # If you change any of these options, adjust ConsolidatedLoginsController::DeviseMappingShunt accordingly
     devise :database_authenticatable, :registerable, :recoverable, :rememberable, :validatable,
-           :invitable, :trackable, :password_has_required_content
+           :invitable, :trackable
 
     has_many :requests, class_name: 'Partners::Request', foreign_key: :partner_id, dependent: :destroy, inverse_of: :user
     has_many :submitted_partner_requests, class_name: 'Partners::Request', foreign_key: :partner_user_id, dependent: :destroy, inverse_of: :partner_user
     has_many :submitted_requests, class_name: 'Request', foreign_key: :partner_user_id, dependent: :destroy, inverse_of: :partner_user
 
     belongs_to :partner, dependent: :destroy
+
+    validate :password_complexity
+
+    def password_complexity
```

Comment:
Yay! Much better than using a gem :) I am surprised that there isn't a default devise setting for this.